tests/e2e/Pages/MapComponent.ts

```diff
@@ -188,6 +192,29 @@ class MapComponent extends DashboardPage {
     await expect(legendComponent).toBeVisible();
   }
 
+  async assertLegendElementIsNotVisible({
```

Comment:
this new method is not actually used anywhere. Instead the version without 'Not' in the name is used, which is probably better any way for intuitive interpretation.

Reply:
I removed it. It was created with different scope in mind but since we check for element to be visible I had to replace it with the old function. Thanks!